CondFormats/DTObjects/test/stubs/DTDeadUpdate.cc

```diff
@@ -24,7 +24,7 @@ namespace edmtest {
 
   DTDeadUpdate::DTDeadUpdate(int i) : dSum(0) {}
 
-  DTDeadUpdate::~DTDeadUpdate() {}
+  DTDeadUpdate::~DTDeadUpdate() { delete dSum; }
```

Comment:
Hi @namapane 
couldn't https://github.com/cms-sw/cmssw/blob/801a84be47308da449329c459fb4be80103a18d3/CondFormats/DTObjects/test/stubs/DTDeadUpdate.h#L31
be just
```DTDeadFlag dSum;```
and then we don't need to deal with it being a pointer?

Reply:
It could be like this, but I don't find it would be better (and would require more changes). That's anyhow just a test...